packages/slate-analytics/__tests__/index.test.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-process-env */
```

Comment:
This is being referenced a few times throughout the codebase - we could probably add this to the list of rules in `.eslintrc`.

Reply:
Don't want to apply this rule globally. In general I feel like this rule is helpful -- it makes you question you reallllly need to use process-env or not.

Reply:
Fair enough.